webpack.shared.config.js

```diff
@@ -36,6 +36,11 @@ const loaders = [
   {
     test: fileLoaderTest,
     loader: 'file'
+  },
+  {
+    test: /\.jsx?$/,
+    loader: 'standard',
```

Comment:
Ah... though the `.jsx` extension...

You _can_ stack up loaders. `loader1!loader2`